rpm/nghttp2.spec

```diff
@@ -0,0 +1,106 @@
+Summary: Experimental HTTP/2 client, server and proxy
+Name: nghttp2
+Version: 1.41.0
+Release: 1
+License: MIT
+URL: https://nghttp2.org/
+Source0:    %{name}-%{version}.tar.bz2
+
+BuildRequires: c-ares-devel
```

Comment:
Let's swap these to pkgconfig() for each which have pc file.

Reply:
Agree.